source/features/conversation-activity-filter.css

```diff
@@ -1,20 +1,30 @@
 .rgh-conversation-activity-is-filtered .rgh-conversation-activity-filtered,
-.rgh-conversation-activity-is-collapsed-filtered .rgh-conversation-activity-collapsed,
+.rgh-conversation-activity-is-collapsed-filtered :is(
+	.rgh-conversation-activity-collapsed,
+	.js-resolvable-timeline-thread-container[data-resolved='true'], /* Resolved review thread */
+	.js-resolvable-timeline-thread-container[data-resolved='false'] .minimized-comment, /* Minimized review thread comment */
+),
 .rgh-conversation-activity-is-filtered .rgh-conversation-activity-filter-dropdown .octicon-eye,
 .js-issues-results:not(.rgh-conversation-activity-is-filtered) .rgh-conversation-activity-filter-dropdown .octicon-eye-closed {
 	display: none !important;
 }
 
-.rgh-conversation-activity-filter-dropdown .SelectMenu-modal {
-	width: max-content;
-}
-
 /* Debugging-only class. Drop wow to enable it */
-wow:root:root :is(.rgh-conversation-activity-filtered, .rgh-conversation-activity-collapsed) {
+wow:root:root :is(
+.rgh-conversation-activity-is-filtered .rgh-conversation-activity-filtered,
+.rgh-conversation-activity-is-collapsed-filtered :is(
+	.rgh-conversation-activity-collapsed,
+	.js-resolvable-timeline-thread-container[data-resolved='true'], /* Resolved review thread */
+	.js-resolvable-timeline-thread-container[data-resolved='false'] .minimized-comment, /* Minimized review thread comment */
+)) {
 	display: block !important;
 	outline: solid 5px #f00;
```

Comment:
Since it's not CSS only, I think you can just move these 2 properties and the related comment to the main selector, commented out